wremnants/muon_selections.py

```diff
@@ -97,7 +97,7 @@ def select_z_candidate(df, mass_min=60, mass_max=120):
 
     return df
 
-def apply_triggermatching_muon(df, dataset, muon_eta, muon_phi, otherMuon_eta=None, otherMuon_phi=None):
+def apply_triggermatching_muon(df, dataset, muon_eta, muon_phi, otherMuon_eta=None, otherMuon_phi=None, era = "2016PostVFP"):
```

Comment:
I think era must be used somewhere here to get the right trigger

Reply:
Should be fixed now.